include/alpaka/dev/DevGenericSycl.hpp

```diff
@@ -181,10 +182,22 @@ namespace alpaka::trait
             auto find64 = std::find(warp_sizes.begin(), warp_sizes.end(), 64);
             if(find64 != warp_sizes.end())
                 warp_sizes.erase(find64);
```

Comment:
Btw, in C++20, this should be just `std::erase(warp_sizes, 64)`. Looking forward to the upgrade :)